pelican/__init__.py

```diff
@@ -327,6 +327,11 @@ def parse_arguments():
                         help=('Exit the program with non-zero status if any '
                               'errors/warnings encountered.'))
 
+    parser.add_argument('--logs-deduping-min-level', default='WARNING',
+                        choices=('DEUBG', 'INFO', 'WARNING', 'ERROR'),
+                        help=('Only enable logs deduping for logs whose level'
+                              ' is equal or above this value'))
```

Comment:
Suggested help text: _Only enable log de-duplication for levels equal to or above the specified value_

Reply:
Better ! Done